### DISCUSSION TITLE
New member

### DISCUSSION THREAD

**Discussion by idowujoshua1** [Cyfrin/foundry-full-course-cu#2960] [General]
Hey guys,. I'm new here, wish me luck

**Comment by EngrPips** [comment]
Hello buddy, Welcome aboard, Friend.  We wish you the best of luck.

**Reply by idowujoshua1** [reply]
Thanks

**Comment by wakzamilli** [comment]
me is new here......... lol

**Reply by EngrPips** [reply]
Welcome aboard. We wish you the best.

**Reply by wakzamilli** [reply]
arigato sempai

**Comment by chainflash** [comment]
Hello welcome I wish you success.

**Comment by vivek-ux** [comment]
Hi guys I have recently finished 2hours of the first section, so I am new here, lets connect on whatsapp ?? should I create a whatsapp group ???

**Reply by idowujoshua1** [reply]
Go ahead and create, that would serve as our local community to challenge ourselves.

**Reply by Rahul251952** [reply]
definitely buddy.

**Reply by vivek-ux** [reply]
Hi,, I have created a Discord server for the same :- https://discord.gg/KgPerXEx

**Reply by vivek-ux** [reply]
a discord server, https://discord.gg/KgPerXEx

**Comment by nblhjnsr** [comment]
Hi everyone. I'm new here too :)

**Reply by EngrPips** [reply]
Welcome aboard, we wish you the best.

**Comment by davidbasseya** [comment]
Nice, welcome on board and Goodluck, started in February, took about 7 months break busy at other endeavors... i'm fully back in now, still watching the course

**Reply by EngrPips** [reply]
Welcome aboard and I wish you quick recovery
